### DISCUSSION TITLE
Disable automatic "page" generation for every markdown file

### DISCUSSION THREAD

**Discussion by wojtek-rz** [doxygen/doxygen#11178] [Q&A]
**Describe the bug**
I have a project with many `readme.md` files describing the modules.
I would like the content of `readme.md` file to be a description of the current directory,
but I don't want to use "\page" and "\subpage" system, because they are tideous to write and 
I can't include list of all the files in the directory directly on the page.

The content of `readme.md` located in "some/path/to/module/" is in my project:
```
@dir some/path/to/module
Module description...
```
and the problem is that in generated documentation there are **multiple empty "readme" pages**,
because before `@dir` keyword there is nothing. I should be able to disable this automatic "page" 
generation of markdown files.

Or how to make some markdown files to document directories? 

I don't want to use `.dox` files,  as they involve some boilerplate code and can't be easily 
displayed like `.md` files. Maybe some translation in filtering?

**Screenshots**
- **lots of readme pages**
![image](https://github.com/user-attachments/assets/6c509ac3-1d38-4aed-ab8d-e9191947fbe7)
- **empty page**
![image](https://github.com/user-attachments/assets/bd983c98-aaec-441e-844f-b7251ade6076)

**To Reproduce**
This directory contains simple example of Doxygen, that will output empty markdown page.
[doxygen-issue.zip](https://github.com/user-attachments/files/17333915/doxygen-issue.zip)

**What should I do?**
I want to use markdown files and document directories easily.

**Version**
1.9.8

**Comment by albert-github** [comment]
I think this question is about the same issue as reported in question #11176 
Furthermore because there are besides the readme.md file no other files the directory is not show.

Closing it as duplicate of #11176
